### PR TITLE
Adding tests for Cast op

### PR DESCRIPTION
**Pull request by pk-g**
Adding tests for Cast op

### CODE REVIEW

File: onnx/backend/test/case/node/cast.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            input = np.random.random_sample(shape).astype(type_map[getattr(TensorProto, from_type)])
+            node = onnx.helper.make_node(
+                'Cast',
+                inputs=['input'],
+                outputs=['output'],
+                to=to_type
+            )
+            output = input.astype(type_map[getattr(TensorProto, to_type)])
+            expect(node, inputs=[input], outputs=[output],
+               name='test_cast_' + from_type + '_to_' + to_type)
```
Comment (houseroad) [inline onnx/backend/test/case/node/cast.py]: newline character is missing, and indent should be 4 space.

File: onnx/backend/test/case/node/cast.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    @staticmethod
+    def export():
+        shape = (3, 4)
+        test_cases = [
+            ('FLOAT','FLOAT16'),
+            ('FLOAT','DOUBLE'),
+            ('FLOAT16','FLOAT'),
+            ('FLOAT16','DOUBLE'),
+            ('DOUBLE','FLOAT'),
+            ('DOUBLE','FLOAT16')
```
Comment (houseroad) [inline onnx/backend/test/case/node/cast.py]: nit: one more comma

File: onnx/backend/test/case/node/cast.py (hunk 1 above)
Comment (pk-g) [reply onnx/backend/test/case/node/cast.py]: Sure, fixed.

File: onnx/backend/test/case/node/cast.py (hunk 2 above)
Comment (pk-g) [reply onnx/backend/test/case/node/cast.py]: Sure, fixed.

**Comment by bddppq** [comment]
@onnxbot test this please

**Comment by pk-g** [comment]
@bddppq & @houseroad : I'll create another PR for this operation.

**Pull request closed without merging**